src/main/java/nextstep/ladder/domain/LineColumns.java

```diff
@@ -0,0 +1,35 @@
+package nextstep.ladder.domain;
+
+import nextstep.ladder.domain.generator.RandomBooleanGenerator;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class LineColumns {
+
+    private final List<LineRows> columns;
+
+    public LineColumns(List<LineRows> columns) {
+        this.columns = columns;
+    }
+
+    public static LineColumns create(int height, int userCount) {
```

Comment:
```suggestion
    public static LineColumns create(int columnLength ? , int rowLength ? ) {
```

`height`, `userCount`은 `LineColumns.creaete`를 호출하는 입장에서 바라본 이름이 아닐지 생각이 되었어요.
`LineColumns` 객체가 바라보는 관점?, 객체 내부에서 사용하는 관점? 을 좀 더 고려해보면 어떨까요? 😄

Reply:
넵 이것도 앞에서 말씀해주신부분과 동일한거 같네요
외부사용이 아닌 내부사용 관점으로 재작성 해보겠습니다 🙇

Reply:
applied cid: 406afe776c2e7e0e3947d219ed858192aadd3302